app/services/resumeio.py

```
@@ -12,6 +13,16 @@
 
 from app.schemas.resumeio import Extension
 
+# Configure Tesseract path for Windows
+if os.name == 'nt':  # Windows
```

Ruff (0.11.9)

Single quotes found but double quotes preferred

Replace single quotes with double quotes

(Q000)